sgx-tools/verify-quote.go

```diff
@@ -13,24 +13,20 @@ import (
 	"os"
 )
 
-var getIasReportCommand = cli.Command{
-	Name:  "get-ias-report",
-	Usage: "get remote attestation report from IAS",
+var verifyQuoteCommand = cli.Command{
+	Name:  "verify-quote",
+	Usage: "verify quote with the help of IAS",
```

Comment:
Because this is sgx-tools, it can do more things than the others. The generation of IAS report will be still kept in attestation but Enclave TLS will always use quote v2 to support EPID-based attestation and embed quote v2 into TLS cert extension instead of IAS report. The only consumer of IAS report in attestation package will be sgx-tools and shelter. So please keep it.

Reply:
I mean the eventual results would be:
- the verification to quote v2 (EPID) is available
- the verification to quote v3 (ECDSA) is available
- the verification to IAS report is available